code/server/dbscripts/testmatches.sql

```diff
@@ -0,0 +1,5 @@
+insert into user_matches (liker_id, likee_id, likes, createdAt, updatedAt) values (202, 11, 1,  '2016-02-01 03:22:27.833 +00:00', '2016-02-01 03:22:27.833 +00:00');
```

Comment:
After discussion with Jorell we're gonna leave it as it is since deadline is near, will likely revisit in next iteration.